### ISSUE TITLE
Data Table Support in GraphWidgetType

### ISSUE THREAD

**Issue by Laxenade** [cdklabs/cdk-monitoring-constructs#550]
### Feature scope

Custom

### Describe your suggested feature

It looks like the data table type is missing from `GraphWidgetType`. I'm not sure if the data table is a recent widget that CW added or if there hasn't been a use case for it.

https://github.com/cdklabs/cdk-monitoring-constructs/blob/598dea1bd09df69d7d8dbc070ae9728f8629bd67/lib/common/widget/types.ts#L9-L15

It doesn't seem to be too hard to add. I can probably do a PR. I just wanted to confirm the scope first.

**Comment by Laxenade** [comment]
`TableWidget` was added in cdk-lib 5 months ago, so it's relatively recent.

https://github.com/aws/aws-cdk/commit/4599aa3c4502e5e5e5fb7434913f7ce9a6468547

**Comment by Laxenade** [comment]
That's fair. I don't have a strong need for it; I was just playing around with different widgets.

**Comment by echeung-amzn** [comment]
Yes, this should technically be feasible at this point. Feel free to take a stab at it!
